main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
```

Comment:
Your imports must be ordered.

Reply:
Can you explain please?

Reply:
![image](https://user-images.githubusercontent.com/58142395/218115691-30788a35-e845-476d-ab9c-57610f42b1dc.png)
https://peps.python.org/pep-0008/